rdkit/SimDivFilters/UnitTestSimilarityPickers.py

```diff
@@ -0,0 +1,114 @@
+# $Id$
```

Comment:
This isn't really a duplicate. The code in SimilarityPicker implements lazy evaluation. This is triggered either by `len(x)` or by `x[i]`. The doctest covers only one of the two cases.